build-system/tasks/storybook/index.js

```diff
@@ -32,14 +32,11 @@ async function storybookAmp() {
   // install storybook-specific modules
   installPackages(__dirname);
 
-  execScriptAsync(
-    `./node_modules/.bin/start-storybook -c ./amp-env ${storybookArgs}`,
-    {
-      'stdio': [null, process.stdout, process.stderr],
-      cwd: __dirname,
-      env: process.env,
-    }
-  );
+  exec(`./node_modules/.bin/start-storybook -c ./amp-env ${storybookArgs}`, {
+    'stdio': [null, process.stdout, process.stderr],
+    cwd: __dirname,
+    env: process.env,
```

Comment:
You might be able to eliminate the need for all these options. Suggested refactor, in case it works:

**Top-level:**
```js
const storybookCmd = 'build-system/tasks/storybook/node_modules/.bin/start-storybook';
```

**Inside each task:**
```js
const envDir = './amp-env';
const cmd = `${storybookCmd} -c ${envDir} ${storybookArgs}`;
exec(cmd);
```

This might also let you refactor the execution into a single helper function that takes `envDir`.

Reply:
But that doesn't help with `cwd` though, no?

Reply:
All `gulp` tasks change the cwd to the repo root dir. I edited the suggestion to include paths relative to that dir. WDYT?

Reply:
The issue is that we want the `cwd` to be the `build-system/tasks/storybook` instead of the root, so that the dependencies resolve with no need to rewrite them (otherwise for example we'll need to import `./build-system/tasks/storybook/node_modules/@storybook/addon-knobs` instead of `@storybook/addon-knobs`

Reply:
I'll refactor it into a function though

Reply:
Aha, gotcha. Carry on.